File: CaesarCipher/Python/caesarcipher.py
def get_encrypted_message(word): 

    alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ "
    cipher_index = 0
    encrypt = ""

    for user_index in range(len(word)): 

        for alphabet_index in range(len(alphabet)):

            if word[user_index].lower() == alphabet[alphabet_index].lower():
                cipher_index = (alphabet_index + 3) % 27
                encrypt += alphabet[cipher_index]           

    return encrypt


def get_decrypted_message(word): 

    alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ "
    decipher_index = 0
    decrypt = ""

    for user_index in range(len(word)): 

        for alphabet_index in range(len(alphabet)):

            if word[user_index].lower() == alphabet[alphabet_index].lower():
                decipher_index = (alphabet_index - 3 + 27) % 27
                decrypt += alphabet[decipher_index]     

    return decrypt

File: CaesarCipher/Java/CaesarCipher.java
public class CaesarCipher {

    public static String getEncryptedMessage(String userInput) {

        String letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ ";
        char [] alphabet = letters.toCharArray();
        char [] word = userInput.toCharArray();

        char [] encrypt = new char [word.length];
        int cipherIndex = 0;
        int count = 0;

        for(int userIndex = 0; userIndex < word.length; userIndex++) {

            for(int alphabetIndex = 0; alphabetIndex < alphabet.length; alphabetIndex++) {

                if(Character.toLowerCase(word[userIndex]) == Character.toLowerCase(alphabet[alphabetIndex])) {
                    cipherIndex = (alphabetIndex + 3) % 27;
                    encrypt[count] = alphabet[cipherIndex];
                    count++;          
                }

            }

        }

        String result = new String(encrypt);
        return result;

    }



    public static String getDecryptedMessage(String userInput) {


        String letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ ";
        char [] alphabet = letters.toCharArray();
        char [] word = userInput.toCharArray();

        char [] decrypt = new char [word.length];
        int decipherIndex = 0;
        int count = 0;

        for(int userIndex = 0; userIndex < word.length; userIndex++) {

            for(int alphabetIndex = 0; alphabetIndex < alphabet.length; alphabetIndex++) {

                if(Character.toLowerCase(word[userIndex]) == Character.toLowerCase(alphabet[alphabetIndex])) {
                    decipherIndex = (alphabetIndex - 3 + 27) % 27;
                    decrypt[count] = alphabet[decipherIndex];
                    count++;          
                }

            }

        }

        String result = new String(decrypt);
        return result;

    }

}

